pkg/kubectl/cmd/scale.go

```diff
@@ -31,7 +31,7 @@ import (
 
 var (
 	scaleLong = templates.LongDesc(i18n.T(`
-		Set a new size for a Deployment, ReplicaSet, Replication Controller, or Job.
+		Set a new size for a Deployment, ReplicaSet, Replication Controller, or StatefulSet.
```

Comment:
Custom resources can also be scaled now, maybe add them to the list as well? :)

https://deploy-preview-7439--kubernetes-io-vnext-staging.netlify.com/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#scale-subresource

Reply:
> Custom resources can also be scaled now, maybe add them to the list as well? :)

I don't think so, since we can't predict what can and can't be scaled without discovery information. 
 Our help that mentions specific resources is getting further and further out of date.

@soltysh probably worth a mailing list thread at least to discuss options in the general area